### PR TITLE
Sidebar fix

### PR DESCRIPTION
**Pull request by kaytcampbell**
Sidebar will now only collapse when you click the top level li.

### CODE REVIEW

File: app/assets/stylesheets/ama_layout/layout_components/sidebar.scss [hunk 1]
```diff
@@ -32,6 +32,22 @@
...
+      opacity: 1;
+      margin-top: 0;
+
+      .sub-nav:active,
+      .sub-nav:focus{
+        text-decoration: underline;
+      }
+    }
+
+    .waffles{
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/sidebar.scss]: Was this just a testing class?

**Comment by kathleenAMA** [comment]
:hankey: Just a question about the waffles class

**Comment by kaytcampbell** [comment]
:wave: forgot to take out the test class ha

**Comment by kathleenAMA** [comment]
:art: :art: :art: / :elephant: :elephant: :elephant:

**Pull request merged**